lms/djangoapps/courseware/tests/factories.py

```diff
@@ -1,131 +1,30 @@
-# Factories are self documenting  # lint-amnesty, pylint: disable=missing-module-docstring
-
-
+"""
+Build courseware-centric test factories
```

Comment:
TODO: Consider adding a note pointing people to `common/djangoapps/student/tests/factories.py` for LMS-agnostic factories.